### DISCUSSION TITLE
SUGGESTION - hiding electrification poles or using track overlay insted of them (duplicate from Steam)

### DISCUSSION THREAD

**Discussion by gralewski** [voxeltycoon/issues#976] [General]
Hey, it would be super useful to be able to **hide all the electrification poles** in order to get the clutter out. Should be fairly simple to implement because Alt mode can be used and it's just turning rendering of this things off.

You can add an option to **switch into overlay mode during electrification placement** (blue color) to see what is electrified and what is not.

Still - my general feeling is - when I build already very complicated track layout - i just want to avoid tedious clicking and adding clutter to it all. **It's very complicated already and I don't want to complicate it visually more.**

I would just **electrify all tracks in the world in most simple way** because it's downward compatible (just click electrify all button) and just forget about it. I personally don't want to see electrification poles etc. I just like how tracks look. If we enter electrification era we stay in it, so **all tracks should be electrified automatically during laying them on the ground**.

If your idea about the game is to develop other kinds of tracks that are not compatible with each other please remember that **games get boring if we are forced to do tedious things instead of creative things**. Track type changes could be fun and easy - make mass changes in a simple way and it's done.

**Creative part is track layout**, boring part is clicking on already built tracks. Think about this - we have to lay tracks, then add signalling, then add electrification.

Electrification is in my opinion already very boring and tedious. Please be kind to think about quality of life changes that help with boring stuff. Mass electrification could also help.

If I think how many hours I put already into track layout... going all over again through it is just scary.

**Comment by andrewpey** [comment]
Thanks for this insight, we will keep this in mind!
